### ISSUE TITLE
FedRAMP automation

### ISSUE THREAD

**Issue by Ebutton79** [ombegov/cloud.cio.gov#7]
Representative Connolly’s proposed Federal Risk and Authorization Management Program Reform Act would encourage automation in the FedRAMP process. Having worked for one of the country’s largest retailers in a roll involving third-party vendor management, the increased dependency on third-party vendors typically distills security, quality control, and budget management best practices. Continuous monitoring is commonplace, but back-end program management strategies quickly become ineffective without a robust vetting process model that incorporates manual oversight and feedback loops. While it is common to not concentrate all efforts and resources within one entity or framework, minimal use of third-party vendors is recommended.

**Comment by akaak** [comment]
The FedRAMP vetting and authorization process, particularly Joint Authorization Board (JAB)  (https://www.fedramp.gov/jab-authorization/) is well laid out. There could be improvements made to improve the feedback loops. @Ebutton79 suggestion of involving third-party vendors would help the overall process.

**Comment by Ebutton79** [comment]
Right, best practices vs. best (written) policies; definitely a worthy distinction.
